components/clp-package-utils/clp_package_utils/general.py

```
@@ -556,3 +560,21 @@ def validate_path_for_container_mount(path: pathlib.Path) -> None:
                 f" in the container."
             )
+
+
+def validate_dataset(db_config: Database, dataset: str) -> None:
+    """
+    Checks if `dataset` currently exists in the metadata database.
+    :param clp_config:
+    :param dataset:
+    :raise: ValueError
+    """
+    db_config: Database = clp_config.database
```

Flake8 (7.2.0)

[error] undefined name 'clp_config'

(F821)